Split CC overview (#127)
iExecBlockchainComputing/documentation: for-developers/go-to-production.md

# Go to production

> With the integration of SCONE in iExec, you do not need to worry about [remote attestation](intel-sgx-technology.md#remote-attestation). We do that for you, we guarantee that the code is running inside an enclave.

Reviewer: broken link on `remote attestation` after cleanup in `intel-sgx-technology.md`

Reviewer: ```suggestion
With the integration of SCONE in iExec, you do not need to worry about [remote attestation](../help/glossary.md#remote-attestation). We do that for you, we guarantee that the code is running inside an enclave.
```

> All secrets are pushed to an SMS where it is securely saved protected by an enclave, which means even we, the root-privilege user, cannot access it. Only applications you have authorized can get this key.

Reviewer: `saved` or `protected` ? We have `securely saved protected` here.

Reviewer 2: "securely saved, protected by an enclave". This may be confusing but I guess it is correct.

Reviewer: ```suggestion
All secrets are pushed to an SMS where they are securely saved, protected by an enclave, which means even we, the root-privilege user, cannot access them. Only applications you have authorized can get this key.
```
What does `this key` refer to here ?

Reviewer: Do we mean `these secrets` ?

Reviewer 2: I guess we do mean `these secrets`, yes. @jeremyjams ?

Author: Yes

Author: Updated thanks
